### PR TITLE
Update internal app following #20

### PR DESCRIPTION
**Pull request by minwoox**
Motivation:
We do not support username, password verification anymore.

Modification:
- Remove the CLI parameter which takes username

Result:
- No more username, password verification

### CODE REVIEW

File: internal/app/dogma/go.mod [hunk 1]
```diff
@@ -4,7 +4,6 @@ require (
 	github.com/fhs/go-netrc v1.0.0
 	github.com/urfave/cli v1.20.0
 	go.linecorp.com/centraldogma v0.0.0-20181220074123-455505acb33f
```
Comment (linxGnu) [inline internal/app/dogma/go.mod]: :+1: Please update with latest version of central dogma with http/2. I think it's: `a046020` 😄

Comment (minwoox) [reply internal/app/dogma/go.mod]: Oops, fixed. Thank you~!

**Comment by linxGnu** [comment]
LGTM

**Comment by linxGnu** [comment]
I will merged it now. Thank you very much @minwoox well done!

**Pull request merged**